### PR TITLE
Update compat.rst

### PR DESCRIPTION
**Pull request by PaulAngus**
add notice of hypervisors to be deprecated in future releases

### CODE REVIEW

File: source/compat.rst [hunk 1]
```diff
@@ -90,3 +90,25 @@ best results, one of the following browsers recommended:
 -  Google Chrome version 36.0+
 
 -  Safari 6+
+
+Notice Of Management OSes and Hypervisors to be Deprecated
+----------------------------------------------------------
+
+The following hypervisors will no longer be tested from the first release AFTER
+1st June 2018. And will be removed from the CloudStack codebase from the first
```
Comment (DaanHoogland) [inline source/compat.rst]: Do we give any warranty about a number of releases *between* June 1st and Novermber 1st?!!!

File: source/compat.rst [hunk 2]
```diff
@@ -90,3 +90,25 @@ best results, one of the following browsers recommended:
...
+
+The following hypervisors will no longer be tested from the first release AFTER
+1st June 2018. And will be removed from the CloudStack codebase from the first 
+release after 1st November 2018.
+
+-  XenServer 6.2
+-  XenServer 6.5
+-  vSphere 5.0
+-  vSphere 5.1
+-  Ubuntu 12.04
```
Comment (DaanHoogland) [inline source/compat.rst]: How is Ubuntu in the Hypervisor list? should it have a notice about 'with KVM'?

Comment (PaulAngus) [reply source/compat.rst]: Actually, Ubuntu 12.04 only applies to baremetal it seems so can be dropped from this list

File: source/compat.rst (hunk 1 above)
Comment (PaulAngus) [reply source/compat.rst]: No, there is no real way to know what versions will be out in that time right now.

**Comment by PaulAngus** [comment]
yes @rhtyd - 4.11 best to give people as much notice as possible :)

**Comment by rhtyd** [comment]
Okay @PaulAngus changed the base branch to 4.11.

**Comment by rhtyd** [comment]
@DaanHoogland are you lgtm on this?

File: source/compat.rst (hunk 1 above)
Comment (DaanHoogland) [reply source/compat.rst]: of course can we add that remark? something like "There is no way of telling if there will be a version out between those two dates."

**Comment by pdion891** [comment]
LGTM,  we can't tell the upcomming release numbers anyway.

**Comment by DaanHoogland** [comment]
ok, I am fine as is, but a remark that we don't know would be fine

**Pull request merged**